i18n-ui/src/Views/Sample/index.js

```diff
@@ -127,7 +130,7 @@ export default function SamplePage() {
 						{/*
 						  * Sample radio group
 						  */}
-						<FormControl color="secondary" variant="outlined" class={"controlGroup"}>
+						<FormControl color="secondary" variant="outlined" className={styles.controlGroup}>
```

Comment:
import the controlGroupCSS in the function like `const classes = controlGroupCSS();` and then use it like `classes={{ root: classes.root }}`

Reply:
fixed with !important in the module.css